src/net/sourceforge/kolmafia/objectpool/EffectPool.java

```diff
@@ -320,6 +320,7 @@ public class EffectPool {
   public static final int ENHANCED_INTERROGATION = 2415;
   public static final int BOXING_DAY_BREAKFAST = 2429;
   public static final int WOLF_FORM = 2449;
+  public static final int FIDOXENE = 2520;
```

Comment:
If we are trying to sort effects by ID this is in the wrong place.

Reply:
Thanks! Brain fart